scripts/test/Abins/AbinsCalculateSPowderTest.py

```diff
@@ -87,6 +88,10 @@ def _good_case(self, name=None):
 
         self._check_data(good_data=good_data["S"], data=calculated_data.extract())
 
+        # Uncomment to write a replacement data file
```

Comment:
Do we still want this comment?

Reply:
I guess it can go now. I do wish it was a bit easier to regenerate data, but this isn't a great way of managing it.